docs/changelog/2382.feature.rst

```diff
@@ -0,0 +1 @@
+On Windows, PROGRAMFILES, PROGRAMFILES(X86), and PROGRAMDATA environment variables are now passed through.
```

Comment:
Maybe you could mention why this is a good thing?

Reply:
Done in 3eff20c7.